.github/workflows/dockerfiles/alpine

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.14.0
 
-RUN apk add m4 g++ make pkgconf git tar xz gmp-dev re2-dev sqlite-dev fuse-dev ncurses-dev dash sqlite-static ncurses-static linux-headers jq
+RUN apk add m4 g++ make pkgconf git tar xz gmp-dev re2-dev sqlite-dev fuse-dev ncurses-dev dash sqlite-static ncurses-static linux-headers jq postgresql
```

Comment:
Since we are building manually all these should be deleted

Reply:
I think a lot of them are needed, just not the postgres dep

Reply:
oh that is what I meant lol

Reply:
"all these" -> postgres from all these different files not the entire dep list